includes/class-llms-blocks.php

```diff
@@ -137,6 +147,32 @@ public function init() {
 
 	}
 
+	/**
+	 * Load l10n files.
+	 *
+	 * The first loaded file takes priority.
+	 *
+	 * Files can be found in the following order:
+	 *      WP_LANG_DIR/lifterlms/lifterlms-blocks-LOCALE.mo
+	 *      WP_LANG_DIR/plugins/lifterlms-blocks-LOCALE.mo
+	 *
+	 * @since [version]
+	 *
+	 * @return void
+	 */
+	public function load_textdomain() {
+
+		// Load locale.
+		$locale = apply_filters( 'plugin_locale', get_locale(), 'lifterlms' );
+
+		// Load from the LifterLMS "safe" language directory if a translation file exists.
+		load_textdomain( 'lifterlms', WP_LANG_DIR . '/lifterlms/lifterlms-blocks' . $locale . '.mo' );
+
+		// Load localization files.
+		load_plugin_textdomain( 'lifterlms', false, LLMS_BLOCKS_PLUGIN_DIR . '/i18n' );
```

Comment:
```suggestion
		load_plugin_textdomain( 'lifterlms', false, basename( LLMS_BLOCKS_PLUGIN_DIR ) . '/i18n' );
```
because the 3rd parameter of `load_plugin_textdomain` is the 

> Relative path to WP_PLUGIN_DIR where the .mo file resides.
https://developer.wordpress.org/reference/functions/load_plugin_textdomain/

Also, let's consider two things:
1) since the textdomain is `lifterlms` the localization files in LLMS_BLOCKS_PLUGIN_DIR . '/i18n' must be name like `lifterlms-{locale}.mo` and not lifterlms-blocks-{locale}.mo' as one might expect
2) if there's a lifterlms-{locale}.mo in `WP_LANG_DIR . '/plugins/'` (and it's reasonable that the user downloaded the language pack from wp.org) it will win, see:
https://core.trac.wordpress.org/browser/tags/5.5.1/src/wp-includes/l10n.php#L883